docs/authentication/jwt-auth.html

```diff
@@ -16,13 +16,407 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="../../css/reset.css">
     <link rel="stylesheet" href="../../css/docs.css">
+    <link rel="stylesheet" media="screen and (max-width: 900px)" href="../../css/docs_mobile.css">
     <link href=“https://fonts.googleapis.com/css?family=IBM+Plex+Sans” rel=“stylesheet”>
   </head>
   <body>
-    <h1 class="heading-1">JWT Authentication</h1>
+      <main>
+      <h1 class="heading-1">JSON Web Token Authentication</h1>
+      <p class="block-text">A JSON Web Token (JWT) defines a compact and self-contained way for securely transmitting information between parties as a JSON object. You can find out more about JWTs at `JWT.io`.</p>
+      <p class="block-text">This guide uses JWT with Swift-JWT to implement Single Sign On (SSO) authentication for your Kitura routes.</p>
```

Comment:
This is the first time you've mentioned Swift-JWT, this needs an explanation and probably a link to the github project.  This would also be a good place to describe in more detail what we are going to demonstrate in the guide with Swift-JWT and why?